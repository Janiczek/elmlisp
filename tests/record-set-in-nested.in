(set-in myRecord abc
  (set-in anotherRecord def 123))
