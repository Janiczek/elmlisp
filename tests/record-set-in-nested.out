{ myRecord | abc = { anotherRecord | def = 123 } }
